iosApp/iosApp/ViewModels/ContentViewModel.swift

```diff
@@ -45,6 +51,10 @@ class ContentViewModel: ObservableObject {
         }
     }
 
+    @MainActor func loadFeaturePromos() async {
+        featurePromosPending = await (try? featurePromoUseCase.getFeaturePromos()) ?? []
```

Comment:
suggestion: Maybe worth a test that these are loaded as expected

Reply:
I mean, I added [a test](https://github.com/mbta/mobile_app/pull/654/files#diff-797b98f92f1d0e257b46b509e4852a85f686c4dabaf371fa21420d77e3559724) that these are loaded as empty; there are no features to promote (and there is no version before `Nothing` was added, since the version numbers are unsigned) so the list can only ever be empty right now.

Reply:
Oh sorry, I totally overlooked that test somehow - that looks great to me!